charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/node-exporter.rules.test.yaml

```diff
@@ -26,6 +26,16 @@ tests:
     values: '95+0x120'
   - series: 'node_nf_conntrack_entries_limit'
     values: '100+0x120'
+  # PodsRequestCPUApproachNodeCapacity
```

Comment:
Just for the record, the unit tests are explained here: https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/ and can be executed with

```
make test-prometheus
```

In the current state of this PR, the unit tests fail, but please see my previous comment first to make sure that the alert matches the intent of the PR.

Furthermore, the generated file that lists the alerts has to be regenerated by executing the make generate command or rather preferably executing the hack/generate-monitoring-docs.sh script directly to specifically regenerate only the monitoring docs to save some time.

Reply:
Hi Cathy, please focus on @istvanballok's comments that pertain to the alert definition itself. I wanted to further explain why these unit tests fail:
The first reason is what Istvan also mentioned: Your expression evaluates to 1/1=1. The threshold for the alert was set to 80, when it should be 0.8 to denote the percentage value you picked.
The other reason is that the alert wouldn't actually ever fire:

```
  - series: 'kube_pod_container_resource_requests_cpu_cores{node="node1"}'
    values: '1+0x100'
```

means that the value `1` is present for 100 cycles. Because the "scrape" interval defined in L7 is `30s`, that means the value `1` is only present for 50 minutes. The alert however is defined to fire if the condition were true for 1h. After 50 minutes, the value isn't present anymore and becomes stale after 5 minutes. So you would have to change the `values` to `1+0x120` to have test data for one hour.
Lastly, because the alert uses `sum by (node)` it will contain the node label, which isn't present on the expected alert.